dashboard-ui-v2/src/App.tsx

```diff
@@ -31,6 +32,12 @@ async function fetcher<T>(url: string, init?: RequestInit): Promise<T> {
   return res.json()
 }
 
+self.MonacoEnvironment = {
+  getWorker() {
+    return new editorWorker()
+  },
+}
```

Comment:
If this is not present, there will be a warning in console


[docs](https://github.com/suren-atoyan/monaco-react#:~:text=NOTE%3A%20you%20should%20be%20aware%20that%20this%20may%20require%20additional%20webpack%20plugins%2C%20like%20monaco%2Deditor%2Dwebpack%2Dplugin%20or%20it%20may%20be%20impossible%20to%20use%20in%20apps%20generated%20by%20CRA%20without%20ejecting%20them.)